src/Castle.Facilities.WcfIntegration.Tests/Client/WcfClientFixture.cs

```diff
@@ -267,6 +267,9 @@ public void CanResolveClientAssociatedWithChannelUsingSuppliedModel()
 		}
 
 		[Test]
+#if DEBUG
+		[Ignore("This test fails in Debug / relies on GC.Collect to clean weak reference")]
```

Comment:
This is somewhat worrying that `GC.Collect` no longer does what it says on the tin, we've never had a problem with this in the past. Any idea what has changed?